### PR TITLE
Add tron staking sigining

### LINKED ISSUE #129: [ TRON ] Add staking transaction

### LINKED ISSUE #132: [ TRON ] Add Delegator to voting list
Currently we're not adding to the voting list, but updating on each staking action. We should get the current votes and add the delegator to the list.

### LINKED ISSUE #133: [ TRON ] Remove delegator from voting list when unstake.
Unfreeze balance and revote without the selected delegator.

### PR DESCRIPTION
**Pull request by leoneparise**
Fixes #129 
Fixes #132 
Fixes #133

### CODE REVIEW

File: src/app/coins/coin-provider-config.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { CoinType } from "@trustwallet/types";
 import BigNumber from "bignumber.js";
 
 export enum StakeAction {
-  STAKE,
+  STAKE = 0,
```
Comment (dpereskokov) [inline src/app/coins/coin-provider-config.ts]: It equals to 0 by default. I'm not sure it's crucial :)

File: src/app/shared/components/staking/staking.component.ts [hunk 2]
```diff
@@ -101,7 +100,7 @@ export class StakingComponent implements OnInit, OnDestroy {
   getMax(): Observable<{ min: BigNumber; normal: BigNumber }> {
     return combineLatest([this.dataSource.getBalance(), this.config]).pipe(
       map(([balance, config]) => {
-        const additional = CosmosUtils.toAtom(new BigNumber(config.fee));
```
Comment (dpereskokov) [inline src/app/shared/components/staking/staking.component.ts]: oh, missed this one during reactoring

File: src/app/coins/coin-provider-config.ts (hunk 1 above)
Comment (leoneparise) [reply src/app/coins/coin-provider-config.ts]: Yeah. Let me changed that.

File: src/app/shared/components/staking/staking.component.ts (hunk 2 above)
Comment (leoneparise) [reply src/app/shared/components/staking/staking.component.ts]: No worries. I already fixed this.

**Comment by leoneparise** [comment]
I'm still getting an error when I try to broadcast a transaction

**Pull request merged**